research/delf/delf/python/datasets/google_landmarks_dataset/README.md

```diff
@@ -72,7 +72,7 @@ datasets, and not required for the GLDv2 retrieval/recognition datasets.
 Run query feature extraction as follows:
 
 ```bash
-# From models/research/delf/delf/python/google_landmarks_dataset
+# From models/research/delf/delf/python/datasets/google_landmarks_dataset
 python3 ../delg/extract_features.py \
```

Comment:
for this and other commands in this README, I think we need to replace "../delg" by "../../delg" ?